packages/dev/core/src/Rendering/fluidRenderer/fluidRenderer.ts

```diff
@@ -16,16 +16,7 @@ import { FluidRenderingObjectParticleSystem } from "./fluidRenderingObjectPartic
 import { FluidRenderingTargetRenderer } from "./fluidRenderingTargetRenderer";
 import { FluidRenderingObjectCustomParticles } from "./fluidRenderingObjectCustomParticles";
 import { FluidRenderingDepthTextureCopy } from "./fluidRenderingDepthTextureCopy";
-
-import "../../Shaders/fluidRenderingParticleDepth.vertex";
-import "../../Shaders/fluidRenderingParticleDepth.fragment";
-import "../../Shaders/fluidRenderingParticleThickness.vertex";
-import "../../Shaders/fluidRenderingParticleThickness.fragment";
-import "../../Shaders/fluidRenderingParticleDiffuse.vertex";
-import "../../Shaders/fluidRenderingParticleDiffuse.fragment";
-import "../../Shaders/fluidRenderingBilateralBlur.fragment";
-import "../../Shaders/fluidRenderingStandardBlur.fragment";
-import "../../Shaders/fluidRenderingRender.fragment";
+import { ShaderLanguage } from "core/Materials/shaderLanguage";

```

Comment:
I think we use relative import paths ("../../Materials/shaderLanguage"), but I don't know if that changes anything... cc @RaananW

Reply:
I checked to fix this, but there's already too much code using absolute paths, so I'm not sure if this is really a problem

![image](https://github.com/user-attachments/assets/40798781-221d-463c-ac24-5b06c355286b)

Reply:
it's fine to use both, they are being processed the same. As long as you import from a file directly.